### PR TITLE
Switch from CoffeeScript to ES6.

### PR DESCRIPTION
**Pull request by SamirTalwar**
CoffeeScript is now pretty much dead and node.js now supports ES6 out of
the box. Let's use it.

Of course, if anyone is using CoffeeScript, this will work exactly the same.

### CODE REVIEW

**Review by sleepyfox — APPROVED** [review]
I'm sorry to see CS go, but I'm not going to argue at this point.

**Comment by sleepyfox** [comment]
Addition: everything looks kosher.

**Comment by SamirTalwar** [comment]
I prefer CoffeeScript too, but it's refusing to evolve and real-world usage is on the downturn.

… PureScript? :-D

**Comment by sleepyfox** [comment]
I'd be happy with PureScript but probably not very useful for this repo.

N.

File: node/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "microservice-logging",
-  "version": "1.0.0",
+  "version": "1.1.0",
```
Comment (sleepyfox) [inline node/package.json]: Is this not a 'breaking change'?
v2.0.0

**Comment by SamirTalwar** [comment]
Nah, it's not even a minor change. All code will still work exactly as-is.

File: node/package.json (hunk 1 above)
Comment (SamirTalwar) [inline node/package.json]: Nah, it's not even a minor change. All code will still work exactly as-is.

**Pull request merged**